File: app/src/main/java/buggy/plandroid/com/plandroidbugs/room/PGDatabase.java
package buggy.plandroid.com.plandroidbugs.room;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.Room;
import android.arch.persistence.room.RoomDatabase;
import android.content.Context;

@Database(
        version = 2, // Last changed by Aggretsuko
        entities = {
                UserEntity.class,
        }
        )
public abstract class PGDatabase extends RoomDatabase {

    public abstract UserDao userDao();
}

File: app/src/main/java/buggy/plandroid/com/plandroidbugs/room/PGDatabase.kt
package buggy.plandroid.com.plandroidbugs.room

import androidx.room.Database
import androidx.room.RoomDatabase

@Database(
    version = 1,
    entities = [
        UserEntity::class
    ]
)
abstract class PGDatabase : RoomDatabase() {
    abstract fun userDao(): UserDao
}